### ISSUE TITLE
Compiling on stable needs some love

### ISSUE THREAD

**Issue by mbr** [phil-opp/rust-bit-field#2]
```
error: #[feature] may not be used on the stable release channel [E0554]
 --> /home/marc/.cargo/registry/src/github.com-1ecc6299db9ec823/bit_field-0.5.0/src/lib.rs:1:1
  |>
1 |> #![feature(const_fn)]
  |> ^^^^^^^^^^^^^^^^^^^^^

```

Pretty please? =)

**Comment by phil-opp** [comment]
We don't even need const functions anymore :D.

Fixed in 5fee893b7649c584b30d8e446cee977ba9169292

**Comment by phil-opp** [comment]
Ok, it compiles on stable now. Published as 0.6.0.

**Comment by mbr** [comment]
❤
